README.md

```diff
@@ -25,26 +25,67 @@ Dashboard for [Apache APISIX](https://github.com/apache/apisix)
 
 The goal of the Apache APISIX Dashboard project is to enable everyone to quickly experience and learn Apache APISIX, and it cannot be used directly in a production environment. Therefore, its function point coverage is always a subset of Apache APISIX, and may lag behind the rapid iteration of Apache APISIX. If you need to use the Dashboard project in a production system, you need to enhance user permissions, communication security, high availability and advanced features.
 
-## User Guide
+## Install
 
-Please refer to [User Guide](./docs/USER_GUIDE.md)
+We have multiple ways to install APISIX dashboard
```

Comment:
For technical documents, try to avoid using "we" or "you". It is more common to start with a verb.

For this line, here is an example: `Supports multiple ways to install APISIX dashboard`

You can update other similar points in this way.

Reply:
ok, but in some other project , also can see `we` or `you`, more kindly